ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+"""
+ndmg.stats.qa_skullstrip
+~~~~~~~~~~~~~~~~~~~~~~
+The top level ndmg quality assurance for skull strip module.
+In this module, ndmg:
+1. Input original t1w file and the skull-striped brain file
+2. Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)
+3. save the image into output directory
```

Comment:
`saves`, not `save`

Reply:
fix it

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/1dd9c2ea28f04426eeebed4d5e78b2b171ff0731